### PR TITLE
📖 Update helm chart repo docs

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

Following from PR #2209

### CODE REVIEW

File: site/docs/v1.3.0-beta.1/customize-installation.md [hunk 1]
```diff
@@ -12,6 +12,14 @@
   - [Generate YAML only](#generate-yaml-only)
   - [Additional options](#additional-options)
   - [Optional Velero CLI configurations](#optional-velero-cli-configurations)
+    - [Enabling shell autocompletion](#enabling-shell-autocompletion)
+  - [Bash on Linux](#bash-on-linux)
```
Comment (skriss) [inline site/docs/v1.3.0-beta.1/customize-installation.md]: I think "Bash on Linux", "Bash on macOS" and "Autocompletion on Zsh" entries should be nested under "Optional Velero CLI configurations"

Comment (ashish-amarnath) [reply site/docs/v1.3.0-beta.1/customize-installation.md]: thanks for catching that. The toc was auto generated based on the markdown headers. fixed the headers.

File: site/docs/master/customize-installation.md [hunk 2]
```diff
@@ -12,6 +12,14 @@
...
   - [Additional options](#additional-options)
   - [Optional Velero CLI configurations](#optional-velero-cli-configurations)
+    - [Enabling shell autocompletion](#enabling-shell-autocompletion)
+    - [Bash on Linux](#bash-on-linux)
+      - [Install bash-completion](#install-bash-completion)
+      - [Enable Velero CLI autocompletion for Bash on Linux](#enable-velero-cli-autocompletion-for-bash-on-linux)
+    - [Bash on macOS](#bash-on-macos)
+      - [Install bash-completion](#install-bash-completion-1)
+      - [Enable Velero CLI autocompletion for Bash on macOS](#enable-velero-cli-autocompletion-for-bash-on-macos)
+      - [Autocompletion on Zsh](#autocompletion-on-zsh)
```
Comment (skriss) [inline site/docs/master/customize-installation.md]: "Autocompletion on Zsh" should be at the same level as the two Bash ones, not nested under "Bash on macOS" (same for section header there) :)

Comment (ashish-amarnath) [reply site/docs/master/customize-installation.md]: didn't follow the
> same for section header there

part. Can you please clarify?

Comment (skriss) [reply site/docs/master/customize-installation.md]: meaning remove a "#" from the "Autocompletion on Zsh" section header

Comment (ashish-amarnath) [reply site/docs/master/customize-installation.md]: done. markdown is hard :-|

Comment (skriss) [reply site/docs/master/customize-installation.md]: yep, this stuff is tedious :/

**Comment by carlisia** [comment]
Hm, sorry. Just now I thought to look at this as a md file. Check this out:

image

I think that header should be removed, because it is not like the other items. Keep the text under it since it's the intro of the section though.

**Comment by carlisia** [comment]
Alternatively:

s/Optional Velero CLI configurations/Enabling shell autocompletion

**Comment by ashish-amarnath** [comment]
I think it should be like 
image

Adding an extra `#` to all headers under `Enabling shell autocompletion`

**Comment by carlisia** [comment]
I don't think there should be a bullet with only 1 sub-bullet but nbd. This def looks better, thank you.

**Pull request merged**